src/context/HistoryContext.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { createContext, useContext, useReducer } from "react";
+import { initialHistory, HistoryReducer } from "../reducer/HistoryReducer";
+
+const defaultHistoryValue = {};
+
+const HistoryContext = createContext(defaultHistoryValue);
```

Comment:
Can you please comment here, what was the use of "deafultHistoryValue" variable?

Reply:
It is not useful here but I was following the syntax so defined it. Will refactor it! thanks for pointing it out